x-pack/plugins/cases/common/types/api/configure/v1.test.ts

```diff
@@ -26,6 +27,20 @@ describe('configure', () => {
       connector: serviceNow,
       closure_type: 'close-by-user',
       owner: 'Cases',
+      customFields: [
```

Comment:
Let's add a test where you can create a configuration without custom fields (or the opposite remove this code and create a test where we can add custom fields).